sdk/remoterendering/tests.yml

```diff
@@ -3,6 +3,7 @@ trigger: none
 extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
+    UseFederatedAuth: true
```

Comment:
One thing noticed when migrating remote-rendering to az-json was – there are tests that uses Storage Sas key, and those tests failed due to the new access restriction placed on the storage resources in general. 

There is a [documentation](https://learn.microsoft.com/en-us/azure/remote-rendering/how-tos/create-an-account#link-storage-accounts) using role assignment for rendering resource to access storage, so Sas can be avoided. Though for azure-json I used Sas.

Reply:
That's fine, we will move them to aux tenants later on, will keep them as is.

Reply:
thanks, sounds good.